src/core.js

```diff
@@ -70,7 +70,7 @@ jQuery.fn = jQuery.prototype = {
 
 	// Take an array of elements and push it onto the stack
 	// (returning the new matched element set)
-	pushStack: function( elems ) {
+	pushStack:function(elems ) {
```

Comment:
format: style mismatch:
```suggestion
	pushStack: function( elems ) {
```

`489e0fea` ␣ should be inserted at column 12.
`55a8147f` ␣ should be inserted at column 21.